src/components/calendar/ClassModal.js

```diff
@@ -26,6 +28,21 @@ function ClassModal({ showDialog, toggleDialog, classes, section, removeSection
         <Typography variant="h5">
           {`${section.subjectId} ${section.courseId} Section ${section.sectionNumber}`}
         </Typography>
```

Comment:
Let's put a [Divider](https://v3-9-0.material-ui.com/demos/dividers/) here. If it looks too cramped, make sure there's a healthy amount of space after the divider (before the rest of the dialog content) too.